lib/probes/v8.js

```diff
@@ -1,103 +1,115 @@
-var v8 = require('v8');
-var utils = require('../utils/module');
+var v8 = null;
+try {
+  v8 = require('v8');
```

Comment:
can't we use `semver` here ? It will be better than using a try/catch no ?

Reply:
Done.